src/Layout/redist/targets/OverlaySdkOnLKG.targets

```diff
@@ -9,9 +9,19 @@
     <Exec Command="$(DotnetTool) --version" ConsoleToMsbuild="true">
       <Output TaskParameter="ConsoleOutput" PropertyName="Stage0SdkVersion"/>
     </Exec>
-    <!--Prepare Microsoft.DotNet.Common.*.nupkg instead of copying from LKG SDK-->
-    <Exec Command="$(DotnetTool) pack $(RepoRoot)template_feed\Microsoft.DotNet.Common.ProjectTemplates.7.0 --configuration $(Configuration)" />
-    <Exec Command="$(DotnetTool) pack $(RepoRoot)template_feed\Microsoft.DotNet.Common.ItemTemplates --configuration $(Configuration)" />
+
+    <!-- Get templates version -->
+    <ItemGroup>
+      <TemplatesFolderPath Include="$([System.IO.Directory]::GetDirectories(`$(_DotNetHiveRoot)templates`,`*.*`))" />
+      <TemplatesFolderPath>
+        <FolderName>$([System.IO.Path]::GetFileName(`%(Identity)`))</FolderName>
+      </TemplatesFolderPath>
+    </ItemGroup>
+    <Error Text="SDK Stage 0 has more than one folder with templates: @(TemplatesFolderPath->'%(FolderName)')" Condition="@(TemplatesFolderPath->Count()) > 1"></Error>
```

Comment:
Is there a better way to get this version number?

Reply:
This folder is a runtime version that was installed with the stage 0 SDK.
I haven't found other way, it seems it was not used before. 

In fact any version that starts with 7.0 works for testing purpose, so alternatively we can use constant number or SDK stage 0 version; but then copy other templates to that folder too. The templates will be loaded from the version with latest version for given major.minor.